src/backend/marsha/core/serializers.py

```diff
@@ -442,7 +442,6 @@ class XAPIStatementSerializer(serializers.Serializer):
         required=False,
```

Comment:
in your commit message: 
"it will be not be exactly" -> "it will not be exactly"